### PR TITLE
Adding upload capabilities to huntsman-pocs archiver

### PR DESCRIPTION
**Pull request by bazkiaei**

### CODE REVIEW

File: scripts/archive-images.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 from huntsman.pocs.archive.archiver import Archiver
 
 if __name__ == "__main__":
-
```
Comment (lspitler) [inline scripts/archive-images.py]: Please undo this as it serves no purpose.

File: scripts/sync_data_with_remote_server.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        archive_directory=archive_directory,
+        remote_host=remote_host,
+        username=username,
+        password=password,
+        port=port,
+    )
+    archiver.start()
+
+    # Monitor the archiver
+    # If it breaks we want to terminate the script so docker can restart the service
```
Comment (lspitler) [inline scripts/sync_data_with_remote_server.py]: I don't think docker will be doing any restarting of the service. This should run outside of docker.

File: scripts/sync_data_with_remote_server.py [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    while True:
```
Comment (lspitler) [inline scripts/sync_data_with_remote_server.py]: Can you add a `try` that gracefully handles a Keyboardinterrupt. What if this happens during a file transfer?

Comment (bazkiaei) [reply scripts/sync_data_with_remote_server.py]: The introduced class in this file looks for `fits` files and copy them to the destination and delete each file after copying is completed.

Therefore in a scenario like the one you mentioned, the file that its copying is not completed still exist on the control computer and after re-running the code, the code finds it and transfers it to the server. The code deletes the file when the copying is completed.

File: scripts/sync_data_with_remote_server.py (hunk 2 above)
Comment (bazkiaei) [reply scripts/sync_data_with_remote_server.py]: I started working on this by playing with `archive-images.py` and this comment is left there from that file. I will remove it.

File: scripts/sync_data_with_remote_server.py (hunk 3 above)
Comment (bazkiaei) [reply scripts/sync_data_with_remote_server.py]: Oh! I forgot to add the `try` part! will do that in the next commit when receive your comments.

**Comment by bazkiaei** [comment]
Hey @fergusL ! This PR is ready to be reviewed!

**Review by fergusL — APPROVED** [review]
lgtm

**Pull request merged**